applications/sdp/gpio/src/backend/backend.h

```diff
@@ -9,9 +9,9 @@
 
 #include <drivers/gpio/nrfe_gpio.h>
 
-#if !defined(CONFIG_GPIO_NRFE_EGPIO_BACKEND_ICMSG) && \
-	!defined(CONFIG_GPIO_NRFE_EGPIO_BACKEND_MBOX) && \
-	!defined(CONFIG_GPIO_NRFE_EGPIO_BACKEND_ICBMSG)
+#if !IS_ENABLED(CONFIG_GPIO_NRFE_EGPIO_BACKEND_ICMSG) && \
+	!IS_ENABLED(CONFIG_GPIO_NRFE_EGPIO_BACKEND_MBOX) && \
+	!IS_ENABLED(CONFIG_GPIO_NRFE_EGPIO_BACKEND_ICBMSG)
```

Comment:
Why?

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/16592#discussion_r1784363820

Reply:
> [#16592 (comment)](https://github.com/nrfconnect/sdk-nrf/pull/16592#discussion_r1784363820)

Symbols generated for Kconfig bool options are either defined as 1 or undefined. That's why using `IS_ENABLED()` in `#if` statements [is discouraged](https://github.com/zephyrproject-rtos/zephyr/blob/afce5329f772a4306af699c7a06efddf93cadca0/include/zephyr/sys/util_macro.h#L133-L134) (see also https://github.com/zephyrproject-rtos/zephyr/pull/53200).
But reverting this is probably not worth another CI run.

Reply:
I will do it during the next PR.